src/02_spec/07_modules/cdm/dbgregisters.rst

```diff
@@ -0,0 +1,373 @@
+    - Bits [15:0] (least significant bits) of DCR1
+
+  * - 0x02A0
+    - ``DEBUG_CTRL_REG1_1``
+    - Bits [31:16] of DCR1
+
+  * - 0x02A1
+    - ``DEBUG_CTRL_REG2_0``
+    - Bits [15:0] (least significant bits) of DCR2
+
+  * - 0x02A2
+    - ``DEBUG_CTRL_REG2_1``
+    - Bits [31:16] of DCR2
+
+  * - 0x02A3
+    - ``DEBUG_CTRL_REG3_0``
+    - Bits [15:0] (least significant bits) of DCR3
+
+  * - 0x02A4
+    - ``DEBUG_CTRL_REG3_1``
+    - Bits [31:16] of DCR3
+
+  * - 0x02A5
+    - ``DEBUG_CTRL_REG4_0``
+    - Bits [15:0] (least significant bits) of DCR4
+
+  * - 0x02A6
+    - ``DEBUG_CTRL_REG4_1``
+    - Bits [31:16] of DCR4
+
+  * - 0x02A7
+    - ``DEBUG_CTRL_REG5_0``
+    - Bits [15:0] (least significant bits) of DCR5
+
+  * - 0x02B0
+    - ``DEBUG_CTRL_REG5_1``
+    - Bits [31:16] of DCR5
+
+  * - 0x02B1
+    - ``DEBUG_CTRL_REG6_0``
+    - Bits [15:0] (least significant bits) of DCR6
+
+  * - 0x02B2
+    - ``DEBUG_CTRL_REG6_1``
+    - Bits [31:16] of DCR6
+
+  * - 0x02B3
+    - ``DEBUG_CTRL_REG7_0``
+    - Bits [15:0] (least significant bits) of DCR7
+
+  * - 0x02B4
+    - ``DEBUG_CTRL_REG7_1``
+    - Bits [31:16] of DCR7
+
+  * - 0x02B5
+    - ``DEBUG_MODE_REG1_0``
+    - Bits [15:0] (least significant bits) of DMR1
+
+  * - 0x02B6
+    - ``DEBUG_MODE_REG1_1``
+    - Bits [31:16] of DMR1
+
+  * - 0x02B7
+    - ``DEBUG_MODE_REG2_0``
+    - Bits [15:0] (least significant bits) of DMR2
+
+  * - 0x02C0
+    - ``DEBUG_MODE_REG2_1``
+    - Bits [31:16] of DMR2
+
+  * - 0x02C1
+    - ``DEBUG_WC_REG0_0``
+    - Bits [15:0] (least significant bits) of Debug Watchpoint Counter 0
+
+  * - 0x02C2
+    - ``DEBUG_WC_REG0_1``
+    - Bits [31:16] of Debug Watchpoint Counter 0
+
+  * - 0x02C3
+    - ``DEBUG_WC_REG1_0``
+    - Bits [15:0] (least significant bits) of Debug Watchpoint Counter 1
+
+  * - 0x02C4
+    - ``DEBUG_WC_REG1_1``
+    - Bits [31:16] of Debug Watchpoint Counter 1
+
+  * - 0x02C5
+    - ``DEBUG_STOP_REG_0``
+    - Bits [15:0] (least significant bits) of Debug Stop Register
+
+  * - 0x02C6
+    - ``DEBUG_STOP_REG_1``
+    - Bits [31:16] of Debug Stop Register
+
+  * - 0x02C7
+    - ``DEBUG_REASON_REG_0``
+    - Bits [15:0] (least significant bits) of Debug Reason Register
+
+  * - 0x02D0
+    - ``DEBUG_REASON_REG_1``
+    - Bits [31:16] of Debug Reason Register
+
+  * - 0x02D1
+    - ``DEBUG_CFG_REG_0``
+    - Bits [15:0] (least significant bits) of Debug Configuration Register
+
+  * - 0x02D2
+    - ``DEBUG_CFG_REG_1``
+    - Bits [31:16] of Debug Configuration Register
+
+  * - 0x02D3
+    - ``DEBUG_WRITE_UPDATE``
+    - Bit 0 of this register will be used to commit write data
+ 
+
+
+Debug Value Registers (``DEBUG_VALUE_REG*_*``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Address: *see full register map above*
+- Reset Value: 0
+- Access: read-write
+
+The debug value registers (DVRs) are 32-bit special-purpose supervisor-level registers programmed with the watchpoint/breakpoint addresses or data.
```

Comment:
Please mention where the exact specification for these registers can be found and link to it.